data/inputs_raw/tab_cost_rd_mult.csv

```diff
@@ -1,4 +1,5 @@
 my_road_class,my_road_surface,road_class_mult,road_surface_mult,cost_m_1000s_bridge,cost_embed_cv
+fsr,paved,1,1,25,50
```

Comment:
but it is not a paved road.... You can review the DRA and FTEN road layers manually to see the weirdness (many overlapping road layers in that spot and "paved" is mixed in there for some reason). lets just leave it though.

Reply:
I just added that to the csv so the script would work for Fern. I guess mutating the road surface would have been the way to do it. I've never heard of a paved fsr anyways ha.